app/views/items/_paginator.html.erb

```diff
@@ -17,7 +17,7 @@
       </li>
     <% else %>
       <li>
-        <%= link_to to_page(current-1, opts) do %>
+        <%= link_to to_page(current-1, opts), rel: "prev" do %>
           <span aria-hidden="true">&laquo;</span>
```

Comment:
Off topic: Are there other attributes we could use here for accessibility's sake? Hiding the link seems less than ideal

Reply:
I dunno!  My knowledge of aria attributes is very poor, but we could open an issue to investigate?

Reply:
Oh wait, I think that the link itself is not hidden, but the << span text is hidden because it's not important for non-visual navigation?

Reply:
It doesn't hide the link, but the `aria-hidden` attribute makes it so there is nothing for screen readers to read. The Firefox Dev Tools Accessibility tab seems to confirm this for me. The `aria-hidden` attribute isn't applied if the "«" link is disabled, which is inconsistent too.

I think `aria-label` is what we want. So replace `aria-hidden="true"` with `aria-label="Previous Page"` and `aria-label="Next Page"` respectively. Probably want that with the numbered page links too, e.g. `aria-label="Page 3"`.

Reply:
Ah, that makes sense.  Agreed that we want to label them all.  Do you want to do that or do you want me to do that?

Reply:
I'm on it with a separate PR